src/UploadSessionCursor.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Spatie\Dropbox;
+
+class UploadSessionCursor
+{
+    /**
+     * The upload session ID (returned by upload_session/start).
+     *
+     * @var string
+     */
+    public $session_id;
```

Comment:
Renaming to to lowerCamelCase have a downside: I will need to implement a `toArray()` method, since on current way, the object is directly converted to array by guzzlehttp. Are you sure?